chore: add changesets and enter pre-release mode (#2050)
Redocly/redocly-cli: .changeset/tame-pillows-flash.md

---
"@redocly/respect-core": major
"@redocly/openapi-core": major
"@redocly/cli": major
---

> Removed support for the legacy Redocly API registry in favor of our new Reunite platform, providing improved API management capabilities and better integration with our tooling ecosystem. Migrated the `login` and `push` commands to work exclusively with Reunite. Removed the `preview-docs` command as part of our platform modernization. Users should now use the `preview` command instead.

Reviewer: ```suggestion
Removed support for the legacy Redocly API registry in favor of the new Reunite platform. 
Reunite provides improved API management capabilities and better integration with Redocly's tooling ecosystem. 
Migrated the `login` and `push` commands to work exclusively with Reunite. 
Removed the `preview-docs` command as part of platform modernization. 
Use the `preview` command instead.
```
